JS/Lesson26/savas/edit-post-folder/script.js

```diff
@@ -0,0 +1,45 @@
+//-----------------FUNCTION FOR EDIT POSTS-----------------
+
+document.addEventListener("DOMContentLoaded", function() {
+    const urlParams = new URLSearchParams(window.location.search);
+    const postId = urlParams.get('id');
+    
+    fetch(`https://jsonplaceholder.typicode.com/posts/${postId}`)
+    .then(response => response.json())
+    .then(json => {
+        document.getElementById("edited-post").innerText = postId;
+        document.getElementById("edit-title").value = json.title;
+        document.getElementById("edit-body").value = json.body;
+    })
+    .catch(err => console.log(err));
+
+    const form = document.getElementById("edit-post-form");
```

Comment:
```suggestion
    const form = document.getElementById("edit-form");
```

Reply:
That's it! I couldn't find you "minor error" :)
It works Anna, thank you!